about/team/index.html

```diff
@@ -23,7 +23,7 @@ <h2>{% t team.founders %}</h2>
             <div class="col tmember">
                 <div class="row">
                     <div class="col">
-                        <img src="/img/team/poramin.jpg" alt=""/>
+                        <img src="/img/team/poramin.jpg" alt="Picture of Poramin Insom"/>

```

Comment:
We could remove the trailing `/` from this page as well since we are already editing it.

Reply:
fixed in next commit